src/setup.ts

```diff
@@ -1,10 +1,13 @@
-import { getTestBed } from "@angular/core/testing";
```

Comment:
Why not replace this whole file with a call to Analog's `setupTestBed`?
It has all the typical setup to work with or without isolation etc...
 https://github.com/analogjs/analog/blob/0b255202253a418f88717c6e178f0bb179725e3c/packages/vitest-angular/setup-testbed.ts#L22?

Reply:
will do in a different PR